src/nimf/find.nim

```diff
@@ -245,12 +243,9 @@ proc notFoundPrint() =
     if printBuffer.len > 0:
       inc numFailed
       if numFailed > 16384:
-        var output: string
         withLock(printLock):
-          output = when declared(newStringUninit): newStringUninit(printBuffer.len) else: newString(printBuffer.len)
-          moveMem(addr output[0], addr printBuffer.data[0], printBuffer.len)
+          discard stdout.writeChars(printBuffer.data, 0, printBuffer.len)
```

Comment:
Changing here didnt bring any quantifiable speedup, because im not sure when this codepath is triggered, to bench it properly.